src/statbank/api_types.py

```diff
@@ -1,9 +1,8 @@
 from __future__ import annotations
 
+from typing import NotRequired
 from typing import TypedDict
 
-from typing_extensions import NotRequired
```

Comment:
Denne endringen virker ikke med Python 3.10, og du har ikke endret hvilket Python versjoner du støtter under poetry.dependencies seksjonen i pyrpoject-fila.

Reply:
Siden cruft-templaten legger om til å benytte kun 3.11+ for testing, tenker jeg det fornuftige her er å bumpe pyproject.toml.

Reply:
Jupyterlab på "bakken" er framdels på Python 3.10.9, så da dropper du støtte for å laste opp fra "bakken" også.